libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/GovCert.hs

```diff
@@ -40,27 +44,34 @@ govCertSpec ConwayGovCertEnv {..} vs =
           <> foldMap (getNewMembers . pProcGovAction . gasProposalProcedure) cgceCommitteeProposals
       ccCertSpec coldCred =
         assert . member_ coldCred $ lit knownColdCreds
-   in constrained $ \gc ->
+   in constrained $ \ [var|gc|] ->
         caseOn
           gc
           -- ConwayRegDRep
-          ( branch $ \key coin _ ->
-              [ not_ $ member_ key reps
-              , coin ==. lit (cgcePParams ^. ppDRepDepositL)
+          ( branchW 1 $ \ [var|key1|] [var|coin1|] _ ->
+              [ not_ $ member_ key1 reps
+              , coin1 ==. lit (cgcePParams ^. ppDRepDepositL)
               ]
           )
           -- ConwayUnRegDRep
-          ( branch $ \cred coin ->
-              elem_ (pair_ cred coin) deposits
+          ( branchW 2 $ \ [var|cred|] [var|coin2|] ->
+              -- if one is really unlucky, the ConwayGovCertEnv and VState inputs can cause every branch of the caseOn
+              -- to be false. The easies way to prevent this is to choose 1 branch (this one ConwaUnRegDRep)
```

Comment:
```suggestion
              -- to be false. The easiest way to prevent this is to choose 1 branch (this one ConwayUnRegDRep)
```

Reply:
Resolved by removing this code altogether, as per the discussion in the meeting. Thanks anyway!